### PR TITLE
Update README for APICTL commands

### PR DESCRIPTION
**Pull request by BLasan**
## Purpose
Fixes: [wso2/api-manager#223](https://github.com/wso2/api-manager/issues/490)

## Approach
Added README docs for the following commands.

- Delete API Policy
- Get API Policies
- Import API Policy
- Export API Policy

## Related PRs
- https://github.com/wso2/product-apim-tooling/pull/924

### CODE REVIEW

**Comment by wasuradananjith** [comment]
Let's update the README.html here [1] with new commands.

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html

File: import-export-cli/docs/apictl_delete_policy.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## apictl delete policy
+
+Delete Policy
+
+### Synopsis
+
+Delete a Policy from an environment
+
+```
+apictl delete policy [policy type] (--environment <environment-from-which-the-application-should-be-deleted>) [flags]
```
Comment (ashera96) [inline import-export-cli/docs/apictl_delete_policy.md:10]:
```suggestion
apictl delete policy [policy type] (--environment <environment-from-which-the-policy-should-be-deleted>) [flags]
```

File: import-export-cli/docs/apictl_delete_policy.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+apictl delete policy api -n addHeader -e dev
+NOTE: Both the flags (--name (-n), and --environment (-e)) are mandatory.
+```
+
+### Options
+
+```
+  -e, --environment string   Environment from which the Policy should be deleted
+  -h, --help                 help for policy
```
Comment (ashera96) [inline import-export-cli/docs/apictl_delete_policy.md:24]: Is `help for policy` right? Also please make sure to capitalize the start of a sentence. Check in other places too

File: import-export-cli/docs/apictl_delete_policy_api.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+```
+apictl delete policy api -n addHeader -e dev
+NOTE: All the 2 flags (--name (-n) and --environment (-e)) are mandatory.
+```
+
+### Options
+
+```
+  -e, --environment string   Environment to which the API Policy should be deleted
+  -h, --help                 help for api policy
```
Comment (ashera96) [inline import-export-cli/docs/apictl_delete_policy_api.md:24]: api policy --> API Policy

File: import-export-cli/docs/apictl_delete_policy_api.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+apictl delete policy api -n addHeader -e dev
+NOTE: All the 2 flags (--name (-n) and --environment (-e)) are mandatory.
+```
+
+### Options
+
+```
+  -e, --environment string   Environment to which the API Policy should be deleted
```
Comment (ashera96) [inline import-export-cli/docs/apictl_delete_policy_api.md:23]:
```suggestion
  -e, --environment string   Environment from which the API Policy should be deleted
```

File: import-export-cli/docs/apictl_get_policies_api.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+apictl get policies api -e prod --format jsonArray
+NOTE: The flag (--environment (-e)) is mandatory
+```
+
+### Options
+
+```
+  -e, --environment string   Environment of the API Policies to be fetched
+      --format string        Pretty-print api policies using Go Templates. Use "jsonArray" to list all fields
+  -h, --help                 help for rate-limiting
```
Comment (ashera96) [inline import-export-cli/docs/apictl_get_policies_api.md:29]: rate-limiting?

File: import-export-cli/docs/apictl_get_policies_api.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+apictl get policies api [flags]
+```
+
+### Examples
+
+```
+apictl get policies api -e production
+apictl get policies api -e prod --all
+apictl get policies api -e prod -l 10
+apictl get policies api -e prod
```
Comment (ashera96) [inline import-export-cli/docs/apictl_get_policies_api.md:19]: Do we need both lines 16 and 19?

File: import-export-cli/docs/apictl_get_policies_api.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+apictl get policies api -e prod --format jsonArray
+NOTE: The flag (--environment (-e)) is mandatory
+```
+
+### Options
+
+```
+  -e, --environment string   Environment of the API Policies to be fetched
+      --format string        Pretty-print api policies using Go Templates. Use "jsonArray" to list all fields
```
Comment (ashera96) [inline import-export-cli/docs/apictl_get_policies_api.md:28]:
```suggestion
      --format string        Pretty-print API Policies using Go Templates. Use "jsonArray" to list all fields
```

File: import-export-cli/docs/apictl_delete_policy.md (hunk 2 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_delete_policy.md:24]: In other docs, they have started with a simple letter for the definition of `-h`. Should we have to change those too?

File: import-export-cli/docs/apictl_delete_policy_api.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+## apictl delete policy api
+
+Delete API Policy
+
+### Synopsis
+
+Export API Policy from an environment
+
+```
+apictl delete policy api (--name <name-of-the-api-policy> --environment <environment-from-which-the-api-policies-should-be-deleted>) [flags]
```
[the remark this answers is not in the document]
Comment (BLasan) [reply import-export-cli/docs/apictl_delete_policy_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_delete_policy.md (hunk 1 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_delete_policy.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_delete_policy_api.md (hunk 3 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_delete_policy_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_delete_policy_api.md (hunk 4 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_delete_policy_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_get_policies_api.md (hunk 5 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_get_policies_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_get_policies_api.md (hunk 6 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_get_policies_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

File: import-export-cli/docs/apictl_get_policies_api.md (hunk 7 above)
Comment (BLasan) [reply import-export-cli/docs/apictl_get_policies_api.md]: Fixed by [6747c48](https://github.com/wso2/product-apim-tooling/pull/929/commits/6747c48e9ca4c99780c170b77d8ffa8a29bbf7bd)

**Comment by BLasan** [comment]
> Let's update the README.html here [1] with new commands.
> 
> [1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html

Fixed by [d6f6771](https://github.com/wso2/product-apim-tooling/pull/929/commits/d6f6771c12c2e0ace8e0b1dd872e0ec689d55028)

File: import-export-cli/docs/apictl_delete_policy.md (hunk 2 above)
Comment (ashera96) [reply import-export-cli/docs/apictl_delete_policy.md:24]: Seems like this is not applicable

**Review by wasuradananjith — APPROVED** [review]

**Pull request merged**